ruby/command-t/scanner/file_scanner.rb

```diff
@@ -65,7 +80,13 @@ def show_max_files_warning
       end
 
       def wild_ignore
-        VIM::exists?('&wildignore') && ::VIM::evaluate('&wildignore').to_s
+        ::VIM::evaluate('&wildignore').to_s
+      ensure
```

Comment:
This isn't how `ensure` is supposed to be used. It is analogous to the `finally` in a `try`/`catch`/`finally` block in other languages, and is used to perform clean-up irrespective of exceptions; it isn't used to just return a value.

Additionally, I don't believe this `evaluate` call ever will throw an exception, so I don't think this code path is going to get hit.